scripts/core/interactive-article-actions-panel/actions/publish-tab.tsx

```diff
@@ -35,6 +35,7 @@ interface IProps {
     handleUnsavedChanges(): Promise<IArticle>;
     markupV2: boolean;
     onError: (error: IPanelError) => void;
+    setSave: (item: IArticle) => void;
```

Comment:
The name doesn't look good to me. How about we call it `onChange` or `onDataChange`. What happens after the change is not known by this component so it's better not to use naming that assumes it. Even if the assumption is correct at the moment.